### PR TITLE
Add section on Azure custom roles for DE

### PR DESCRIPTION
**Pull request by traci-morrison**
### Proposed changes
Add a section describing how to create custom Azure roles for DE

### Related issues (optional)
https://docker.atlassian.net/browse/ENGDOCS-73

### CODE REVIEW

**Comment by traci-morrison** [comment]
@ivansharamok PTAL.

**Comment by ivansharamok** [comment]
@traci-morrison, I do not see any links to new content at https://deploy-preview-9768--docsdocker.netlify.com/ee/ucp/admin/install/cloudproviders/install-on-azure/ or any new page under ucp/admin/install/cloudproviders path that points to the new content. How should I navigate to this new page?

**Comment by traci-morrison** [comment]
> @traci-morrison, I do not see any links to new content at https://deploy-preview-9768--docsdocker.netlify.com/ee/ucp/admin/install/cloudproviders/install-on-azure/ or any new page under `ucp/admin/install/cloudproviders` path that points to the new content. How should I navigate to this new page?

@ivansharamok You should now be able to see that section here: https://deploy-preview-9768--docsdocker.netlify.com/ee/ucp/admin/install/cloudproviders/install-on-azure-custom/.

**Comment by ivansharamok** [comment]
@traci-morrison LGTM

**Comment by traci-morrison** [comment]
@usha-mandya PTAL.

File: _data/toc.yaml [hunk 1]
```diff
@@ -1336,6 +1336,8 @@ manuals:
           section:
           - path: /ee/ucp/admin/install/cloudproviders/install-on-azure/
             title: Install on Azure
+          - path: /ee/ucp/admin/install/cloudproviders/install-on-azure-custom/
+            title: Custom Azure Roles
```
Comment (usha-mandya) [inline _data/toc.yaml]: Should this be `roles` with a lowercase r?

File: ee/ucp/admin/install/cloudproviders/install-on-azure-custom.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+---
+title: Custom Azure Roles
+description: Learn how to create custom RBAC roles to run Docker Enterprise on Azure.
+keywords: Universal Control Plane, UCP, install, Docker Enterprise, Azure, Swarm
+---
+
+## Overview
+
+This document describes how to create custom roles to deploy Docker Enterprise resources.
```
Comment (usha-mandya) [inline ee/ucp/admin/install/cloudproviders/install-on-azure-custom.md]: Should we add Azure in the overview section for clarity? That is,`This document describes how to create custom Azure roles...`?

**Comment by traci-morrison** [comment]
> Noted some minor suggestions. Looks good otherwise @traci-morrison.

Thanks @usha-mandya. I incorporated your suggested changes. Can you take another look?

**Pull request merged**